lib/db/schema/patch-023-024.sql

```diff
@@ -0,0 +1,175 @@
+-- Add table for storing unverified sessionTokens.
+CREATE TABLE `unverifiedTokens` (
+  tokenId BINARY(32) NOT NULL PRIMARY KEY,
+  tokenVerificationId BINARY(16) NOT NULL,
+  uid BINARY(16) NOT NULL
+) ENGINE=InnoDB;
```

Comment:
Does this table have an index on `(tokenVerificationId, uid)` in order to support the queries below?  I can't see where it is declared.